.cirrus.yml

```
@@ -5,6 +5,7 @@ freebsd_task:
   name: FREEBSD_DEBUG_NTS
   freebsd_instance:
     image_family: freebsd-13-2
+  skip: "changesIncludeOnly('docs/**', 'NEWS', 'UPGRADING', 'UPGRADING.INTERNALS', '**/README.*', 'CONTRIBUTING.md', 'CODING_STANDARDS.md', '.travis.yml', 'travis/**', '.circleci/**')"
```

Would it make sense to include all markdown files?

```suggestion
  skip: "changesIncludeOnly('docs/**', 'NEWS', 'UPGRADING', 'UPGRADING.INTERNALS', '.travis.yml', 'travis/**', '.circleci/**', '**.md')"
```

---

You are right it does. Same with `**.rst` now I think. 

This list comes from the `push.yml` GitHub Actions workflow. I will propose this change in a separate PR, but with changes to `push.yml` as well.